src/init.cpp

```diff
@@ -439,9 +439,9 @@ std::string HelpMessage(HelpMessageMode mode)
     strUsage += HelpMessageOpt("-rpcsslprivatekeyfile=<file.pem>", strprintf(_("Server private key (default: %s)"), "server.pem"));
     strUsage += HelpMessageOpt("-rpcsslciphers=<ciphers>", strprintf(_("Acceptable ciphers (default: %s)"), "TLSv1.2+HIGH:TLSv1+HIGH:!SSLv2:!aNULL:!eNULL:!3DES:@STRENGTH"));
 
+    strUsage += HelpMessageGroup(_("UI Options:"));
```

Comment:
Maybe 'Metrics UI Options:" or "Console Metrics Options:" in case a user thinks its a GUI e.g. QT or an actual interactive interface?

Also mention that these only apply when not running in daemon mode?